restnavigator/halnav.py

```diff
@@ -167,7 +167,13 @@ class Navigator(object):
     '''
 
     @staticmethod
-    def hal(root, apiname=None, default_curie=None, auth=None, headers=None):
+    def hal(root, 
+            apiname=None, 
+            default_curie=None,
+            auth=None, 
+            headers=None, 
+            session=None
```

Comment:
could you add a trailing comma after this last line? if we add more keyword args in the future, we won't have to modify it